app/utils/gtoken.ts

```
@@ -0,0 +1,285 @@
+
+  /**
+   * Returns a cached token or retrieves a new one from Google.
+   *
+   * @param callback The callback function.
+   */
+  getToken(opts?: GetTokenOptions): Promise<TokenData | undefined>;
+  getToken(callback: GetTokenCallback, opts?: GetTokenOptions): void;
+  getToken(
+    callback?: GetTokenCallback | GetTokenOptions,
+    opts = {} as GetTokenOptions,
+  ): void | Promise<TokenData | undefined> {
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)